[2025-10-14T17:10:04.903+0000] {processor.py:157} INFO - Started process (PID=27048) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:10:04.908+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:10:04.915+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:10:04.914+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:10:06.691+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:10:06.642+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:10:06.696+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:10:06.756+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.873 seconds
[2025-10-14T17:10:37.286+0000] {processor.py:157} INFO - Started process (PID=27109) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:10:37.289+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:10:37.292+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:10:37.291+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:10:38.826+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:10:38.778+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:10:38.836+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:10:38.906+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.632 seconds
[2025-10-14T17:11:09.111+0000] {processor.py:157} INFO - Started process (PID=27170) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:11:09.114+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:11:09.118+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:11:09.117+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:11:10.424+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:11:10.396+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:11:10.431+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:11:10.517+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.417 seconds
[2025-10-14T17:11:40.853+0000] {processor.py:157} INFO - Started process (PID=27229) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:11:40.856+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:11:40.860+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:11:40.859+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:11:42.221+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:11:42.200+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:11:42.226+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:11:42.264+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.423 seconds
[2025-10-14T17:12:12.989+0000] {processor.py:157} INFO - Started process (PID=27288) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:12:12.992+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:12:12.994+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:12:12.994+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:12:14.434+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:12:14.409+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:12:14.442+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:12:14.566+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.588 seconds
[2025-10-14T17:12:45.469+0000] {processor.py:157} INFO - Started process (PID=27348) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:12:45.476+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:12:45.481+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:12:45.480+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:12:46.925+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:12:46.897+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:12:46.930+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:12:46.982+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.536 seconds
[2025-10-14T17:13:17.607+0000] {processor.py:157} INFO - Started process (PID=27410) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:13:17.610+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:13:17.613+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:13:17.612+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:13:18.747+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:13:18.719+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:13:18.753+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:13:18.796+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.202 seconds
[2025-10-14T17:13:49.033+0000] {processor.py:157} INFO - Started process (PID=27472) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:13:49.036+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:13:49.039+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:13:49.038+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:13:50.318+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:13:50.293+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:13:50.323+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:13:50.366+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.344 seconds
[2025-10-14T17:14:20.671+0000] {processor.py:157} INFO - Started process (PID=27532) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:14:20.675+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:14:20.678+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:14:20.678+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:14:22.039+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:14:22.018+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:14:22.044+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:14:22.088+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.430 seconds
[2025-10-14T17:14:52.615+0000] {processor.py:157} INFO - Started process (PID=27599) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:14:52.619+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:14:52.624+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:14:52.623+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:14:53.827+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:14:53.804+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:14:53.832+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:14:53.874+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.271 seconds
[2025-10-14T17:15:24.728+0000] {processor.py:157} INFO - Started process (PID=27661) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:15:24.731+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:15:24.734+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:15:24.733+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:15:25.935+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:15:25.909+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:15:25.941+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:15:26.008+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.290 seconds
[2025-10-14T17:15:56.795+0000] {processor.py:157} INFO - Started process (PID=27722) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:15:56.798+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:15:56.801+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:15:56.800+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:15:58.594+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:15:58.528+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:15:58.599+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:15:58.653+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.869 seconds
[2025-10-14T17:16:28.949+0000] {processor.py:157} INFO - Started process (PID=27784) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:16:28.953+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:16:28.960+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:16:28.958+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:16:30.857+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:16:30.836+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:16:30.862+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:16:30.900+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.962 seconds
[2025-10-14T17:17:01.253+0000] {processor.py:157} INFO - Started process (PID=27844) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:17:01.259+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:17:01.263+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:17:01.261+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:17:02.294+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:17:02.272+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:17:02.298+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:17:02.336+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.115 seconds
[2025-10-14T17:17:32.546+0000] {processor.py:157} INFO - Started process (PID=27905) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:17:32.551+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:17:32.556+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:17:32.555+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:17:33.903+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:17:33.882+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:17:33.906+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:17:33.948+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.420 seconds
[2025-10-14T17:18:04.672+0000] {processor.py:157} INFO - Started process (PID=27966) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:18:04.675+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:18:04.678+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:18:04.677+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:18:05.789+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:18:05.771+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:18:05.794+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:18:05.835+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.174 seconds
[2025-10-14T17:18:36.286+0000] {processor.py:157} INFO - Started process (PID=28026) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:18:36.292+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:18:36.298+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:18:36.296+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:18:37.432+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:18:37.404+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:18:37.439+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:18:37.483+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.221 seconds
[2025-10-14T17:19:07.793+0000] {processor.py:157} INFO - Started process (PID=28085) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:19:07.796+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:19:07.799+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:19:07.798+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:19:09.122+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:19:09.104+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:19:09.127+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:19:09.178+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.400 seconds
[2025-10-14T17:19:39.493+0000] {processor.py:157} INFO - Started process (PID=28144) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:19:39.496+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:19:39.508+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:19:39.499+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:19:42.121+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:19:42.065+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:19:42.127+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:19:42.245+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 2.771 seconds
[2025-10-14T17:20:13.036+0000] {processor.py:157} INFO - Started process (PID=28203) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:20:13.039+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:20:13.043+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:20:13.042+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:20:14.373+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:20:14.352+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:20:14.378+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:20:14.420+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.394 seconds
[2025-10-14T17:20:44.980+0000] {processor.py:157} INFO - Started process (PID=28263) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:20:44.987+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:20:44.995+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:20:44.994+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:20:47.197+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:20:47.178+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:20:47.203+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:20:47.249+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 2.292 seconds
[2025-10-14T17:21:18.068+0000] {processor.py:157} INFO - Started process (PID=28323) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:21:18.071+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:21:18.075+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:21:18.074+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:21:19.464+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:21:19.404+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:21:19.469+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:21:19.535+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.477 seconds
[2025-10-14T17:21:49.871+0000] {processor.py:157} INFO - Started process (PID=28383) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:21:49.874+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:21:49.877+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:21:49.876+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:21:51.444+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:21:51.420+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:21:51.449+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:21:51.526+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.665 seconds
[2025-10-14T17:22:21.970+0000] {processor.py:157} INFO - Started process (PID=28442) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:22:21.974+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:22:21.977+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:22:21.976+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:22:23.542+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:22:23.512+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:22:23.547+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:22:23.607+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.650 seconds
[2025-10-14T17:22:53.839+0000] {processor.py:157} INFO - Started process (PID=28501) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:22:53.843+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:22:53.846+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:22:53.845+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:22:54.924+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:22:54.904+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:22:54.929+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:22:54.976+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.149 seconds
[2025-10-14T17:23:25.459+0000] {processor.py:157} INFO - Started process (PID=28560) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:23:25.464+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:23:25.468+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:23:25.467+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:23:26.886+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:23:26.871+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:23:26.889+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:23:26.932+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.489 seconds
[2025-10-14T17:23:58.292+0000] {processor.py:157} INFO - Started process (PID=28628) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:23:58.295+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:23:58.302+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:23:58.299+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:24:00.859+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:24:00.706+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:24:00.864+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:24:00.946+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 2.674 seconds
[2025-10-14T17:24:31.894+0000] {processor.py:157} INFO - Started process (PID=28688) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:24:31.897+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:24:31.900+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:24:31.899+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:24:33.679+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:24:33.657+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:24:33.683+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:24:33.724+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.846 seconds
[2025-10-14T17:25:04.523+0000] {processor.py:157} INFO - Started process (PID=28748) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:25:04.528+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:25:04.535+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:25:04.533+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:25:06.391+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:25:06.370+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:25:06.396+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:25:06.469+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.962 seconds
[2025-10-14T17:25:37.051+0000] {processor.py:157} INFO - Started process (PID=28809) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:25:37.055+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:25:37.058+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:25:37.057+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:25:38.414+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:25:38.396+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:25:38.418+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:25:38.457+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.418 seconds
[2025-10-14T17:26:08.758+0000] {processor.py:157} INFO - Started process (PID=28869) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:26:08.761+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:26:08.765+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:26:08.764+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:26:10.480+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:26:10.452+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:26:10.488+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:26:10.545+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.798 seconds
[2025-10-14T17:26:40.794+0000] {processor.py:157} INFO - Started process (PID=28928) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:26:40.796+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:26:40.798+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:26:40.798+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:26:41.859+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:26:41.836+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:26:41.865+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:26:41.905+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.124 seconds
[2025-10-14T17:27:12.360+0000] {processor.py:157} INFO - Started process (PID=28988) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:27:12.376+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:27:12.388+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:27:12.387+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:27:17.844+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:27:17.397+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:27:17.848+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:27:17.956+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 5.683 seconds
[2025-10-14T17:27:48.592+0000] {processor.py:157} INFO - Started process (PID=29051) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:27:48.596+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:27:48.599+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:27:48.598+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:27:50.194+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:27:50.167+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:27:50.199+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:27:50.245+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.668 seconds
[2025-10-14T17:28:20.748+0000] {processor.py:157} INFO - Started process (PID=29111) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:28:20.752+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:28:20.755+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:28:20.755+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:28:22.854+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:28:22.825+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:28:22.860+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:28:22.913+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 2.177 seconds
[2025-10-14T17:28:53.644+0000] {processor.py:157} INFO - Started process (PID=29171) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:28:53.647+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:28:53.653+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:28:53.652+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:28:55.368+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:28:55.337+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:28:55.375+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:28:55.426+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.798 seconds
[2025-10-14T17:29:25.675+0000] {processor.py:157} INFO - Started process (PID=29231) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:29:25.681+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:29:25.708+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:29:25.707+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:29:26.758+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:29:26.717+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:29:26.767+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:29:26.857+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.196 seconds
[2025-10-14T17:29:57.396+0000] {processor.py:157} INFO - Started process (PID=29290) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:29:57.399+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:29:57.402+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:29:57.401+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:29:58.363+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:29:58.328+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:29:58.369+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:29:58.427+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.046 seconds
[2025-10-14T17:30:29.338+0000] {processor.py:157} INFO - Started process (PID=29349) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:30:29.342+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:30:29.345+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:30:29.344+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:30:31.007+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:30:30.977+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:30:31.012+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:30:31.103+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.791 seconds
[2025-10-14T17:31:01.707+0000] {processor.py:157} INFO - Started process (PID=29417) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:31:01.711+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:31:01.715+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:31:01.713+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:31:02.924+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:31:02.888+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:31:02.930+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:31:03.010+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.320 seconds
[2025-10-14T17:31:33.899+0000] {processor.py:157} INFO - Started process (PID=29478) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:31:33.906+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:31:33.911+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:31:33.909+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:31:35.095+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:31:35.062+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:31:35.100+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:31:35.152+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.264 seconds
[2025-10-14T17:32:05.836+0000] {processor.py:157} INFO - Started process (PID=29538) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:32:05.844+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:32:05.849+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:32:05.847+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:32:06.939+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:32:06.913+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:32:06.946+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:32:06.996+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.179 seconds
[2025-10-14T17:32:37.751+0000] {processor.py:157} INFO - Started process (PID=29599) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:32:37.755+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:32:37.758+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:32:37.757+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:32:38.771+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:32:38.747+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:32:38.775+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:32:38.821+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.089 seconds
[2025-10-14T17:33:09.082+0000] {processor.py:157} INFO - Started process (PID=29660) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:33:09.086+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:33:09.089+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:33:09.088+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:33:10.333+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:33:10.311+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:33:10.337+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:33:10.377+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.309 seconds
[2025-10-14T17:33:40.772+0000] {processor.py:157} INFO - Started process (PID=29720) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:33:40.775+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:33:40.777+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:33:40.777+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:33:41.823+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:33:41.802+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:33:41.831+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:33:41.872+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.112 seconds
[2025-10-14T17:34:12.234+0000] {processor.py:157} INFO - Started process (PID=29782) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:34:12.236+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:34:12.239+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:34:12.238+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:34:13.010+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:34:12.988+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:34:13.015+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:34:13.076+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.851 seconds
[2025-10-14T17:34:43.560+0000] {processor.py:157} INFO - Started process (PID=29841) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:34:43.565+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:34:43.569+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:34:43.568+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:34:44.924+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:34:44.901+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:34:44.929+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:34:44.984+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.443 seconds
[2025-10-14T17:35:15.088+0000] {processor.py:157} INFO - Started process (PID=29901) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:35:15.091+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:35:15.094+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:35:15.093+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:35:16.068+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:35:16.032+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:35:16.072+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:35:16.172+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.094 seconds
[2025-10-14T17:35:46.654+0000] {processor.py:157} INFO - Started process (PID=29963) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:35:46.657+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:35:46.660+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:35:46.659+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:35:47.874+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:35:47.847+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:35:47.879+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:35:47.938+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.297 seconds
[2025-10-14T17:36:19.021+0000] {processor.py:157} INFO - Started process (PID=30025) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:36:19.024+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:36:19.026+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:36:19.026+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:36:20.047+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:36:20.022+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:36:20.052+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:36:20.105+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.091 seconds
[2025-10-14T17:36:50.410+0000] {processor.py:157} INFO - Started process (PID=30084) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:36:50.413+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:36:50.417+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:36:50.416+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:36:52.002+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:36:51.962+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:36:52.008+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:36:52.087+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.689 seconds
[2025-10-14T17:37:22.868+0000] {processor.py:157} INFO - Started process (PID=30143) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:37:22.870+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:37:22.873+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:37:22.872+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:37:23.798+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:37:23.778+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:37:23.803+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:37:23.854+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.993 seconds
[2025-10-14T17:37:54.502+0000] {processor.py:157} INFO - Started process (PID=30202) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:37:54.505+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:37:54.509+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:37:54.508+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:37:55.882+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:37:55.853+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:37:55.886+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:37:55.957+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.467 seconds
[2025-10-14T17:38:26.568+0000] {processor.py:157} INFO - Started process (PID=30261) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:38:26.577+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:38:26.583+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:38:26.583+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:38:28.276+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:38:28.249+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:38:28.280+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:38:28.347+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.791 seconds
[2025-10-14T17:38:58.855+0000] {processor.py:157} INFO - Started process (PID=30321) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:38:58.860+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:38:58.864+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:38:58.863+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:38:59.688+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:38:59.668+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:38:59.692+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:38:59.746+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.911 seconds
[2025-10-14T17:39:30.039+0000] {processor.py:157} INFO - Started process (PID=30380) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:39:30.042+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:39:30.044+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:39:30.043+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:39:30.841+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:39:30.821+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:39:30.845+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:39:30.893+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.864 seconds
[2025-10-14T17:40:01.999+0000] {processor.py:157} INFO - Started process (PID=30440) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:40:02.003+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:40:02.006+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:40:02.006+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:40:03.830+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:40:03.743+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:40:03.838+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:40:03.903+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.922 seconds
[2025-10-14T17:40:34.705+0000] {processor.py:157} INFO - Started process (PID=30500) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:40:34.708+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:40:34.712+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:40:34.711+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:40:36.097+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:40:36.058+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:40:36.104+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:40:36.203+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.509 seconds
[2025-10-14T17:41:07.144+0000] {processor.py:157} INFO - Started process (PID=30569) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:41:07.150+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:41:07.164+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:41:07.163+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:41:09.018+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:41:08.991+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:41:09.024+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:41:09.125+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 2.001 seconds
[2025-10-14T17:41:39.558+0000] {processor.py:157} INFO - Started process (PID=30630) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:41:39.561+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:41:39.563+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:41:39.563+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:41:40.517+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:41:40.492+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:41:40.522+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:41:40.573+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.024 seconds
[2025-10-14T17:42:10.803+0000] {processor.py:157} INFO - Started process (PID=30690) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:42:10.806+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:42:10.809+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:42:10.808+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:42:11.897+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:42:11.870+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:42:11.903+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:42:11.949+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.158 seconds
[2025-10-14T17:42:42.322+0000] {processor.py:157} INFO - Started process (PID=30752) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:42:42.326+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:42:42.329+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:42:42.328+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:42:43.230+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:42:43.207+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:42:43.236+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:42:43.277+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.967 seconds
[2025-10-14T17:43:13.870+0000] {processor.py:157} INFO - Started process (PID=30812) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:43:13.873+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:43:13.877+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:43:13.876+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:43:15.392+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:43:15.346+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:43:15.397+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:43:15.486+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.637 seconds
[2025-10-14T17:43:45.769+0000] {processor.py:157} INFO - Started process (PID=30873) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:43:45.776+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:43:45.782+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:43:45.781+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:43:46.906+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:43:46.878+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:43:46.911+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:43:46.964+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.215 seconds
[2025-10-14T17:44:17.451+0000] {processor.py:157} INFO - Started process (PID=30933) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:44:17.466+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:44:17.470+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:44:17.469+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:44:18.356+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:44:18.329+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:44:18.361+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:44:18.419+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.010 seconds
[2025-10-14T17:44:48.748+0000] {processor.py:157} INFO - Started process (PID=30993) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:44:48.751+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:44:48.755+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:44:48.755+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:44:49.826+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:44:49.795+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:44:49.832+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:44:49.881+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.146 seconds
[2025-10-14T17:45:20.245+0000] {processor.py:157} INFO - Started process (PID=31055) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:45:20.249+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:45:20.253+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:45:20.251+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:45:21.417+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:45:21.377+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:45:21.425+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:45:21.486+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.255 seconds
[2025-10-14T17:45:51.754+0000] {processor.py:157} INFO - Started process (PID=31116) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:45:51.761+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:45:51.766+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:45:51.765+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:45:52.937+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:45:52.914+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:45:52.941+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:45:52.979+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.246 seconds
[2025-10-14T17:46:23.263+0000] {processor.py:157} INFO - Started process (PID=31178) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:46:23.266+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:46:23.269+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:46:23.268+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:46:24.361+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:46:24.343+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:46:24.365+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:46:24.405+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.155 seconds
[2025-10-14T17:46:54.534+0000] {processor.py:157} INFO - Started process (PID=31239) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:46:54.537+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:46:54.540+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:46:54.539+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:46:55.712+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:46:55.682+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:46:55.717+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:46:55.769+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.246 seconds
[2025-10-14T17:47:25.926+0000] {processor.py:157} INFO - Started process (PID=31301) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:47:25.930+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:47:25.932+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:47:25.932+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:47:27.047+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:47:27.024+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:47:27.051+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:47:27.093+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.177 seconds
[2025-10-14T17:47:57.699+0000] {processor.py:157} INFO - Started process (PID=31360) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:47:57.703+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:47:57.709+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:47:57.707+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:47:58.828+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:47:58.794+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:47:58.832+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:47:58.917+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.234 seconds
[2025-10-14T17:48:29.525+0000] {processor.py:157} INFO - Started process (PID=31422) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:48:29.528+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:48:29.531+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:48:29.530+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:48:30.589+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:48:30.554+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:48:30.594+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:48:30.645+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.132 seconds
[2025-10-14T17:49:00.877+0000] {processor.py:157} INFO - Started process (PID=31484) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:49:00.881+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:49:00.884+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:49:00.883+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:49:01.995+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:49:01.971+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:49:02.005+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:49:02.056+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.189 seconds
[2025-10-14T17:49:32.446+0000] {processor.py:157} INFO - Started process (PID=31543) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:49:32.450+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:49:32.454+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:49:32.452+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:49:33.544+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:49:33.518+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:49:33.548+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:49:33.601+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.168 seconds
[2025-10-14T17:50:04.120+0000] {processor.py:157} INFO - Started process (PID=31606) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:50:04.129+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:50:04.138+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:50:04.135+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:50:05.095+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:50:05.063+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:50:05.102+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:50:05.153+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.055 seconds
[2025-10-14T17:50:35.282+0000] {processor.py:157} INFO - Started process (PID=31667) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:50:35.285+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:50:35.288+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:50:35.287+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:50:36.094+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:50:36.068+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:50:36.102+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:50:36.151+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.880 seconds
[2025-10-14T17:51:06.824+0000] {processor.py:157} INFO - Started process (PID=31726) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:51:06.827+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:51:06.830+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:51:06.829+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:51:07.796+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:51:07.769+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:51:07.801+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:51:07.848+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.035 seconds
[2025-10-14T17:51:38.239+0000] {processor.py:157} INFO - Started process (PID=31787) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:51:38.244+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:51:38.250+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:51:38.248+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:51:39.220+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:51:39.193+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:51:39.225+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:51:39.306+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.085 seconds
[2025-10-14T17:57:14.658+0000] {processor.py:157} INFO - Started process (PID=76) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:57:14.661+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:57:14.673+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:57:14.672+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:57:15.995+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:57:15.984+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:57:15.997+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:57:16.031+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.380 seconds
[2025-10-14T17:57:46.552+0000] {processor.py:157} INFO - Started process (PID=136) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:57:46.557+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:57:46.561+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:57:46.561+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:57:47.334+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:57:47.318+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:57:47.337+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:57:47.371+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.826 seconds
[2025-10-14T17:58:17.692+0000] {processor.py:157} INFO - Started process (PID=195) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:58:17.701+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:58:17.718+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:58:17.717+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:58:20.650+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:58:20.628+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:58:20.670+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:58:20.788+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 3.109 seconds
[2025-10-14T17:58:51.337+0000] {processor.py:157} INFO - Started process (PID=254) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:58:51.343+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:58:51.373+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:58:51.372+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:58:54.124+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:58:54.098+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:58:54.128+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:58:54.197+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 2.892 seconds
[2025-10-14T17:59:24.626+0000] {processor.py:157} INFO - Started process (PID=315) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:59:24.628+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:59:24.632+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:59:24.631+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:59:26.055+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:59:26.023+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:59:26.059+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:59:26.153+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.534 seconds
[2025-10-14T17:59:56.419+0000] {processor.py:157} INFO - Started process (PID=375) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:59:56.422+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T17:59:56.428+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:59:56.427+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:59:57.622+0000] {logging_mixin.py:151} INFO - [2025-10-14T17:59:57.608+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T17:59:57.627+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T17:59:57.663+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.254 seconds
[2025-10-14T18:00:27.930+0000] {processor.py:157} INFO - Started process (PID=435) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:00:27.932+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:00:27.937+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:00:27.937+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:00:28.862+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:00:28.841+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:00:28.865+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:00:28.897+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.974 seconds
[2025-10-14T18:00:59.051+0000] {processor.py:157} INFO - Started process (PID=497) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:00:59.053+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:00:59.058+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:00:59.058+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:01:00.947+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:01:00.923+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:01:00.951+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:01:00.999+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.955 seconds
[2025-10-14T18:01:31.505+0000] {processor.py:157} INFO - Started process (PID=556) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:01:31.507+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:01:31.512+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:01:31.511+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:01:32.224+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:01:32.205+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:01:32.227+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:01:32.257+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.759 seconds
[2025-10-14T18:02:02.831+0000] {processor.py:157} INFO - Started process (PID=615) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:02:02.839+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:02:02.854+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:02:02.853+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:02:03.936+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:02:03.920+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:02:03.939+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:02:03.973+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.171 seconds
[2025-10-14T18:02:34.263+0000] {processor.py:157} INFO - Started process (PID=677) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:02:34.266+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:02:34.271+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:02:34.270+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:02:35.277+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:02:35.265+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:02:35.281+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:02:35.321+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.066 seconds
[2025-10-14T18:03:05.965+0000] {processor.py:157} INFO - Started process (PID=739) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:03:05.968+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:03:05.973+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:03:05.973+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:03:07.121+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:03:07.098+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:03:07.125+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:03:07.170+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.221 seconds
[2025-10-14T18:03:38.854+0000] {processor.py:157} INFO - Started process (PID=801) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:03:38.859+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:03:38.884+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:03:38.877+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:03:41.399+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:03:41.381+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:03:41.402+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:03:41.454+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 2.621 seconds
[2025-10-14T18:04:11.527+0000] {processor.py:157} INFO - Started process (PID=860) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:04:11.529+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:04:11.533+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:04:11.532+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:04:12.120+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:04:12.108+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:04:12.123+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:04:12.143+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.621 seconds
[2025-10-14T18:04:42.416+0000] {processor.py:157} INFO - Started process (PID=919) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:04:42.419+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:04:42.431+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:04:42.430+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:04:43.161+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:04:43.133+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:04:43.164+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:04:43.186+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 0.779 seconds
[2025-10-14T18:05:13.710+0000] {processor.py:157} INFO - Started process (PID=980) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:05:13.712+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:05:13.720+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:05:13.719+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:05:14.865+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:05:14.853+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:05:14.868+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:05:14.909+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.220 seconds
[2025-10-14T18:05:45.298+0000] {processor.py:157} INFO - Started process (PID=1041) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:05:45.301+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:05:45.307+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:05:45.306+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:05:46.316+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:05:46.300+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:05:46.320+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:05:46.346+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.058 seconds
[2025-10-14T18:06:16.534+0000] {processor.py:157} INFO - Started process (PID=1100) to work on /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:06:16.537+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/chromedriver_v2.py for tasks to queue
[2025-10-14T18:06:16.545+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:06:16.544+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:06:17.872+0000] {logging_mixin.py:151} INFO - [2025-10-14T18:06:17.857+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/chromedriver_v2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/chromedriver_v2.py", line 6, in <module>
    from py_files.get import ChromebrowserOption, OpenChromeBrowser, StockResults
  File "/opt/airflow/dags/py_files/get.py", line 48, in <module>
    class OpenChromeBrowser:
  File "/opt/airflow/dags/py_files/get.py", line 54, in OpenChromeBrowser
    def open_chrome(self, options, chrome_service = Service(executable_path= r"C:\Users\kzakr\gielda\chromedriver-win64\chromedriver.exe")):
NameError: name 'Service' is not defined
[2025-10-14T18:06:17.875+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/chromedriver_v2.py
[2025-10-14T18:06:17.916+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/chromedriver_v2.py took 1.403 seconds
